CHANGELOG.md

```diff
@@ -11,7 +11,7 @@ We follow the [Semantic Versioning 2.0.0](http://semver.org/) format.
 - **cf-forms:** [PATCH] Fixed checkbox/radio borders on hover
 
 ### Removed
--
+- **capital-framework:** Remove IE7 and below support.
```

Comment:
This should probably be a `[MINOR]` right? It doesn't really change much but feels more than a PATCH